v2/argsparser.go

```diff
@@ -99,9 +108,13 @@ parseLoop:
 			switch {
 			case isIdentInterior(r):
 				tag.WriteRune(r)
+			case r == ':': // allowed in tag names
+				tag.WriteRune(r)
```

Comment:
@thockin We could expand the grammar from `<tagBody> ::= <name> ...` to `<tagBody> ::= { <prefix> ":" } <name> ...`.  But what I did here was to just allow ":" in tag names.

Thoughts welcome.

Reply:
Do we have a use-case for it?

Reply:
We have to somehow support `+k8s:<name>`. My test case for such tags was failing with the grammar.